### PR TITLE
Check if user is member of city

### PR DESCRIPTION
**Pull request by NatalyMinziuk**
dev
## JIRA

* [Main JIRA ticket](https://jira.softserve.academy/secure/RapidBoard.jspa?rapidView=id)

## Code reviewers

- [ ] @github_username

### Second Level Review

- [ ] @github_username

## Summary of issue

ToDo

## Summary of change

ToDo

## Testing approach

ToDo

## CHECK LIST
- [ ]  СI passed
- [ ]  Сode coverage >=95%
- [ ]  PR is reviewed manually again (to make sure you have 100% ready code)
- [ ]  All reviewers agreed to merge the PR
- [ ]  I've checked new feature as logged in and logged out user if needed
- [ ]  PR meets all conventions

### CODE REVIEW

**Review by DimaDiD — APPROVED** [review]

**Review by Daryna-del — APPROVED** [review]

File: EPlast/EPlast.WebApi/Controllers/AdminController.cs [hunk 1]
```diff
@@ -330,5 +330,17 @@ public async Task<IActionResult> RegionsAdmins()
 
             return Ok(model);
         }
+        /// <summary>
+        /// Change user role to expired
+        /// </summary>
+        /// <param name="userId">The id of the user</param>
+        /// <response code="200">Successful operation</response>
+        /// <response code="404">User not found</response>
```
Comment (IlliaDR) [inline EPlast/EPlast.WebApi/Controllers/AdminController.cs:338]: Seems like this ///summary doesn't belong here, does it? Plus you don't return status codes explicitly, which is not ok, i guess. But maybe that's just me.

**Review by DimaDiD — APPROVED** [review]

File: EPlast/EPlast.WebApi/Controllers/AdminController.cs (hunk 1 above)
Comment (NatalyMinziuk) [reply EPlast/EPlast.WebApi/Controllers/AdminController.cs]: fixed

File: EPlast/EPlast.WebApi/Controllers/AdminController.cs [hunk 2]
```diff
@@ -330,5 +330,15 @@ public async Task<IActionResult> RegionsAdmins()
 
             return Ok(model);
         }
+        /// <summary>
+        ///  Check if user is member of city
+        /// </summary>
+        /// <param name="userId">The id of the user</param>
+        [HttpPut("IsCityMember/{userId}")]
```
Comment (IlliaDR) [inline EPlast/EPlast.WebApi/Controllers/AdminController.cs:337]: IMO you should use Get method here instead of Put.

Comment (NatalyMinziuk) [reply EPlast/EPlast.WebApi/Controllers/AdminController.cs]: fixed

**Review by IlliaDR — APPROVED** [review]

**Pull request merged**